pages/trees/[id].tsx

```diff
@@ -82,54 +84,75 @@ const InfoList: FC<{
   nowcastData: NowcastDataType[] | null
   nowcastIsLoading: boolean
   nowcastError: Error | null
-}> = ({ treeData, nowcastData, nowcastIsLoading, nowcastError }) => {
+  rainData: RainDataType[] | null
+  rainIsLoading: boolean
+  rainError: Error | null
+}> = ({ treeData, nowcastData, nowcastIsLoading, nowcastError, rainData }) => {
+  const { t } = useTranslation('common')
   const averageStatusId =
     nowcastData &&
     !nowcastIsLoading &&
     !nowcastError &&
     nowcastData[3].value &&
     mapSuctionTensionToStatus(nowcastData[3].value)?.id
 
+  const totalRain = rainData?.reduce(
```

Comment:
_If_ we do the calculation in the frontend, I would suggest moving this to the `useRainData` hook and returning a `sum` from there. What do you think?

Reply:
We now do the calculation on the API side so this isn't necessary anymore.